pkg/updatestatus/nodeinformer.go

```diff
@@ -194,6 +202,37 @@ func (c *nodeInformerController) sync(ctx context.Context, syncCtx factory.SyncC
 	return nil
 }
 
+func (c *nodeInformerController) initializeCachesNoErrors() {
+	if err := c.initializeCaches(); err != nil {
+		klog.Errorf("Failed to initialize caches: %v", err)
+	}
+}
```

Comment:
I wondered whether we'd need to abort the controller :thinking: 

I _think_ it is safe to continue the controller despite these failures because it just means we started cold and will eventually load all items from the cache?

Reply:
Done.

https://github.com/openshift/cluster-version-operator/pull/1164/commits/a81ec69e31f504ce10a44650cfa5c4d4260fd2cd#diff-c5a41cb83200e13544e0509a5e77c180e16a1d70d7411a79cac2bc107e89950fR99

I am not 100% sure if I got your suggestion correctly and keep this unsolved so it is more visible to you to get another look.